### PR TITLE
[New] Set initial viewpoint

### PR DESCRIPTION
**Pull request by fcamas**
## Description

This PR implements `Set initial viewpoint` in `Maps` category.
URL to [README](https://devtopia.esri.com/runtime/common-samples/tree/main/designs/SetInitialViewpoint)

## Linked Issue(s)

- `swift/issues/`

## How To Test
When the sample loads, note the map is opened at the initial view point that is set to it

## Screenshots
set-initial-viewpoint

### CODE REVIEW

File: Shared/Samples/Set initial viewpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Setting the initial viewpoint is useful when a user wishes to first load the map at a particular area of interest.
+
+## How to use the sample
+
+When the sample loads, note the map is opened at the initial view point that is set to it.
+
+## How it works
+
+1. Instantiate an `Map` object.
```
Comment (pgruenler) [inline Shared/Samples/Set initial viewpoint/README.md:17]:
```suggestion
1. Instantiate a `Map` object.
```

File: Shared/Samples/Set initial viewpoint/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+2. Instantiate a `Viewpoint` object using an `Envelope` object.
+3. Set the starting location of the map with `setInitialViewpoint(Viewpoint)`.
```
Comment (pgruenler) [inline Shared/Samples/Set initial viewpoint/README.md:19]:
```suggestion
3. Set the starting location of the map with `map.initialViewpoint = Viewpoint(boundingGeometry:)`.
```

**Review by pgruenler — COMMENTED** [review]
Looks good. Just a couple comments.

File: Samples.xcodeproj/project.pbxproj.orig
Comment (pgruenler) [inline Samples.xcodeproj/project.pbxproj.orig:1]: Please remove this file. It's an artifact of the merge conflict.

File: Samples.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -140,6 +140,8 @@
 		218F35C229C290BF00502022 /* AuthenticateWithOAuthView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = 218F35B329C28F4A00502022 /* AuthenticateWithOAuthView.swift */; };
 		3E30884A2C5D789A00ECEAC5 /* SetSpatialReferenceView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = 3EEDE7CD2C5D73F700510104 /* SetSpatialReferenceView.swift */; };
 		3EEDE7CE2C5D73F700510104 /* SetSpatialReferenceView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3EEDE7CD2C5D73F700510104 /* SetSpatialReferenceView.swift */; };
+		3E720F9D2C619B1700E22A9E /* SetInitialViewpointView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3E720F9C2C619B1700E22A9E /* SetInitialViewpointView.swift */; };
+		3E720F9E2C61A0B700E22A9E /* SetInitialViewpointView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = 3E720F9C2C619B1700E22A9E /* SetInitialViewpointView.swift */; };
```
Comment (CalebRas) [inline Samples.xcodeproj/project.pbxproj:144]: Could you sort these? Right click on Samples/Shared/Sample folder in project navigator select Sort by Name. (Sometimes they get messed up after merging.)

**Review by CalebRas — COMMENTED** [review]
Looks good and works on all platforms.

File: Samples.xcodeproj/project.pbxproj (hunk 3 above)
Comment (fcamas) [reply Samples.xcodeproj/project.pbxproj:143]: Thank you for the instructions! I sorted them as you suggested

File: Samples.xcodeproj/project.pbxproj.orig
Comment (fcamas) [reply Samples.xcodeproj/project.pbxproj.orig:1]: Thank you for pointing that out! I removed the file.

**Review by CalebRas — APPROVED** [review]

**Review by pgruenler — APPROVED** [review]

**Pull request merged**